dynamic_config/dynamic_config_manager.py

```diff
@@ -0,0 +1,195 @@
+"""Component that handles dynamic configuration of the Envoy proxy."""
```

Comment:
Can we be a bit more specific indicating that this is the Envoy proxy which is the system under test and not the one used by Nighthawk as the test server?